src/timeatlas/read_write/text.py

```diff
@@ -41,6 +46,19 @@ def read_text(path: str) -> Any:
         raise IOError("The path doesn't' include any recognizable files")
 
 
+def read_tsd(path: str) -> TimeSeriesDataset:
```

Comment:
As it's a method that will be directly used by user, provide an explicit docstring. How should the files in read by `read_tsd()` be structured so that it's readable?

Should we make a new issue to clarify this in general?